arm-sql/2014-04-01/examples/FirewallRuleCreateMax.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "firewallrulecrudtest-12",
+    "serverName": "firewallrulecrudtest-6285",
+    "firewallRuleName": "firewallrulecrudtest-5370",
+    "api-version": "2014-04-01",
+    "parameters": {
+      "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/firewallrulecrudtest-12/providers/Microsoft.Sql/servers/firewallrulecrudtest-6285/firewallRules/firewallrulecrudtest-3927",
```

Comment:
properties id, location and kind are being ignored, and marked as read only for Firewall, you can remove this as this has no effect on the creation of firewall, also then I think there won't be 2 variants of this for min and max.